src/backend/renderer.js

```diff
@@ -479,7 +479,7 @@ export function attach(
       case IndeterminateComponent:
         return getDisplayName(resolvedType);
       case EventComponent:
-        return type.displayName || 'EventComponent';
+        return type.responder.displayName || 'EventComponent';
```

Comment:
Will `type` always have a `responder` value or should we add a null check here? (Will this break FB5 if the DT sync lands before React?)

Reply:
It will always have a responder.